src/character.cpp

```diff
@@ -12495,17 +12515,38 @@ int Character::impact( const int force, const tripoint &p )
         effective_force = force + hard_ground;
         mod = slam ? 1.0f : fall_damage_mod();
         if( here.has_furn( p ) ) {
-            // TODO: Make furniture matter
+            if( here.furn( p ) == furn_f_bed ||
+                here.furn( p ) == furn_f_bed_down ||
+                here.furn( p ) == furn_f_down_mattress ||
+                here.furn( p ) == furn_f_floor_mattress ) {
+                effective_force = adjust_effective_force_for_soft_landing( effective_force );
+            } else {
+                //if furniture breakable it breaks and slighly reduces damage
+                if( here.is_bashable_furn( p ) && here.furn( p )->bash.str_max <= effective_force ) {
+                    here.destroy_furn( tripoint_bub_ms( p ), true );
+                    effective_force -= here.furn( p )->bash.str_max * 0.2f;
+                }
+            }
```

Comment:
This could be handled by adding the field in furniture objects and the bashable amount doesn't make any sense.  Plenty of bashable furniture that would actually hurt you more or have potential to hurt you more.

Reply:
I know, was orienting more on table / shelf , soft chair or sofa, as a landing spot for the rest it doesn't really matter probably can be added that falling on safeguards would hurt more?

Reply:
Yeah I think if you do it as a field someone could give specific furnitures positive falling damage.